src/site/content/en/handbook/quick-start/index.md

```diff
@@ -11,7 +11,7 @@ description: |
 Content creation for web.dev has three phases: planning, writing, and publishing.
 
 ## Planning
-1. Start a content proposal issue on GitHub using [this issue template](https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=content+request&template=content_request.md&title=). The checklist in your issue will guide you through the rest of the process.
```

Comment:
Should they read the web.dev audience doc or other materials before coming up with the proposal? This is about planning, so it seems appropriate to point them to some stuff they should read when planning a topic to write about

Reply:
Good point. Will add.

Reply:
Actually, we have links in the proposal template. You think that suffices as a way to keep the number of steps down in this process doc?